db/migrations/20220707114243-create-users.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      firstName: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      lastName: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      avatar: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      email: {
```

Comment:
Can't see login field here - so, I suppose email would be used as login, right?
it should be unique